past.json

```diff
@@ -0,0 +1,32 @@
+---
+layout: none
+---
+[
+{%- assign sorted_conferences = site.conferences | sort: 'date_start' | reverse -%}
+{%- assign today = site.time | date: "%s" -%}
+{%- for conference in sorted_conferences -%}
+  {%- assign date_end = conference.date_end | date: "%s" -%}
+  {%- if today >= date_end %}
+  {
+    "name": "{{ conference.name }}",
+    "website": "{{ conference.website }}",
+    "location": "{{ conference.location }}",
+    {%- if conference.status %}
+    "status": "{{ conference.status }}",
+    {%- endif -%}
+    {%- if conference.online %}
+    "online": true,
+    {%- endif %}
+    "dateStart": "{{ conference.date_start | date: "%Y-%m-%d" }}",
+    "dateEnd": "{{ conference.date_end | date: "%Y-%m-%d" }}",
```

Comment:
This will lead to a trailing comma (that's not allowed in JSON) when there's no CfP information.